build.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
     if (releaseMode) {
         compile 'com.epam.reportportal:commons'
     } else {
-        implementation 'com.github.reportportal:commons:b6926df'
+        compile 'com.github.reportportal:commons:f4c73c1'
```

Comment:
As coppile depricated, please use api to recive accses to transitive dependencies